infra/github-runner.nix

```diff
@@ -7,10 +7,9 @@ in
   imports = [ <nixpkgs/nixos/modules/virtualisation/amazon-image.nix> ];
   ec2.hvm = true;
 
-  services.github-runner = {
+  services.github-runners."$${runner-name}" = {
```

Comment:
Oh, is that a double-level interpolation? :smiling_face_with_tear:

Reply:
We heard you like interpolating, so here's how to interpolate while you interpolate... No, this is actually `$$` escaping the terraform string interpolation, because nix and terraform syntax conflicts. This wouldn't have happened if we could write NixOS configurations in Nickel :stuck_out_tongue: